### ISSUE TITLE
Add PINN mode

### ISSUE THREAD

**Issue by SuperSashka** [ITMO-NSS-team/torch_DE_solver#31]
It is good to have possibility to add something like data=data and term in a loss function to be able to train model in a PINN mode.

**Comment by SuperSashka** [comment]
Здесь хочется всё-таки, чтобы мы могли внутри лосса задавать отдельно настраиваемое слагаемое, которое на основе (опционально) своей сетки и своих данных борется с диффуром в процессе обучения с собственным весом. Соответственно для такого нужны свои функции для вычисления и объединения с текущим лоссом. То есть сделать L=lam_data*L_data+lam_op*L_op+lam_bc*L_bc

**Comment by SuperSashka** [comment]
At the time being we have separate 'data' BC type #36 that has its own weight and may be included into adaptive lambda procedure.

However, I still think that it may be improved further, so it is not closed atm.
